packages/stack/src/components-page/README.md

```diff
@@ -0,0 +1,175 @@
+# Stack SDK Components Documentation
+
+The following documentation provides detailed information about various components and functionalities related to account settings, authentication, team interactions, and more within the Stack SDK. These components are essential for incorporating authentication and account management capabilities into applications using the Stack SDK.
+
+---
+
+## AccountSettings
+
+**Type**: Component
+
+**Description**: This component allows users to manage their account settings, including profile information, password, email verification, and multi-factor authentication (MFA).
+
+**Usage**:
+```jsx
+<AccountSettings fullPage={true} />
+```
+
+**Properties**:
+- `fullPage`: `boolean` - Determines if the settings should be displayed as a full page. Default is `false`.
+
+**Notes**: The component encapsulates several sections, each responsible for a specific part of the user settings, providing a comprehensive management interface.
+
+---
+
+## AuthPage
+
+**Type**: Component
+
+**Description**: A component rendering the authentication page, which can dynamically switch between sign-in and sign-up forms based on the provided props.
+
+**Usage**:
+```jsx
+<AuthPage fullPage={true} type="sign-in" />
+```
+
+**Properties**:
+- `fullPage`: `boolean` - If `true`, the authentication page will occupy the full page.
+- `type`: `'sign-in' | 'sign-up'` - Specifies the type of authentication form to display.
+
+**Notes**: This component also handles redirections based on authentication state and can optionally integrate with social OAuth providers.
+
+---
+
+## EmailVerification
+
+**Type**: Component
+
+**Description**: Manages the email verification process, displaying appropriate messages and handling verification logic.
+
+**Usage**:
+```jsx
+<EmailVerification searchParams={{ code: "verification_code" }} fullPage={true} />
+```
+
+**Properties**:
+- `searchParams`: `{ code: string }` - Query parameters containing the verification code.
+- `fullPage`: `boolean` - Whether the message should be displayed as full page.
+
+---
+
+## ForgotPassword
+
+**Type**: Component
+
+**Description**: Offers users the ability to initiate a password reset procedure via email.
+
+**Usage**:
+```jsx
+<ForgotPassword fullPage={true} />
+```
+
+**Properties**:
+- `fullPage`: `boolean` - Determines if the forgot password component should be displayed as a full page.
```

Comment:
It uses different wordings for this property in different components (eg. here vs. EmailVerification) — would be nice to have consistency